wca-guidelines.md

```diff
@@ -92,6 +92,7 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-5><puzzle-defects><puzzledefects> Article 5: Puzzle Defects
 
+- 5b+) [ADDITION] The competitor is not permitted to cause a puzzle defect intentionally, unless correcting a previous puzzle defect (see [Regulation 5b3b](regulations:regulation:5b3b) and [Regulation 5b3c](regulations:regulation:5b3c)). Examples: removing an attached center cap, rotating a corner piece, peeling a sticker.
```

Comment:
To be consistent with other Guidelines, I think we should change "Examples:" to something like: "Examples of puzzle defects that can be caused intentionally: removing ...". What do you think?

Reply:
Done! Could you review again?

Reply:
LGTM now :)

Reply:
Thanks! 😄